README.md

```diff
@@ -223,6 +225,11 @@ get_regression_points(score_model)
 
 ``` r
 library(broom)
```

Comment:
Might need to update packages?

Reply:
sorry, where? all my local packages are up to date

Reply:
sorry, missed this

Reply:
will update R